question/bank/previewquestion/classes/previewquestion_helper.php

```diff
@@ -144,7 +156,7 @@ public static function question_preview_form_url($questionid, $context, $preview
      * @param object $context
      * @param moodle_url $returnurl
```

Comment:
add the new parameter

Reply:
Parameters are present in doc, not sure about comment's signification